app/javascript/react/components/molecules/FixedStreamStationHeader/atoms/StationActionButtons/StationActionButtons.tsx

```diff
@@ -34,4 +34,9 @@ const StationActionButtons = () => {
   );
 };
 
+const copyCurrentURL = () => {
+  const currentURL = window.location.href;
+  navigator.clipboard.writeText(currentURL);
+};
+
```

Comment:
I suggest moving this function before the return statement. I believe that this is standard here :)